app/uk/gov/hmrc/driverinspectionnotificationfrontend/views/inspectionStatusResults/required/inspection_required_export.scala.html

```diff
@@ -81,6 +82,21 @@ <h3 class="govuk-heading-m">@messages("inspection_needed_export.arriving.h3")</h
         <li><a href="@{appConfig.checkDelaysInlandBorderFacilityUrl}">@messages("inspection_needed_export.arriving.p1.link2")</a></li>
     </ul>
 
+    @govukInsetText(InsetText(
+        content = Text(messages("inspection_needed_export.arriving.inset"))
+    ))
+
+    <p class="govuk-body">@messages("inspection_needed_export.arriving.p2")</p>
+    <ul class="govuk-list govuk-list--bullet" >
+        <li>@messages("inspection_needed_export.arriving.li.1")</li>
+        <li>@messages("inspection_needed_export.arriving.li.2")</li>
+        <li>@messages("inspection_needed_export.arriving.li.3")</li>
+    </ul>
+
+    <p class="govuk-body">
+        @messages("inspection_needed_export.arriving.p3") <a href="https://www.gov.uk/government/publications/attending-an-inland-border-facility/attending-an-inland-border-facility#if-your-vehicle-is-excluded">
```

Comment:
minor: there is a hmrcNewTabLink component that we can use here
you can see it here: https://github.com/hmrc/goods-movement-system-frontend/pull/1017/files#diff-47ba94c3b4a56fad3f538aa6fc029f94324eecd58585a121a7a190e21a2f08a4R25

Reply:
new tab link doesn't work. So still stick to the original.